libsolidity/ast/AsmJsonImporter.cpp

```diff
@@ -104,6 +104,8 @@ yul::Statement AsmJsonImporter::createStatement(Json::Value const& _node)
 		return createContinue(_node);
 	else if (nodeType == "Leave")
 		return createLeave(_node);
+	else if (nodeType == "Block")
```

Comment:
Was this case just missing? Or was it left out on purpose before?

Reply:
It was just missing. And untested. The blocks in ``if``s and in ``for``s are handled in ``createIf`` and ``createFor`` (or whatever they're called) directly - so this is just about cases like:
```
assembly {
  {}
 }
```
Where blocks occur in statement position... and apparently that didn't occur anywhere in any tests...

Reply:
ok